bundles/CoreBundle/src/Resources/config/services.yaml

```diff
@@ -8,3 +8,15 @@ services:
 
     LoyaltyCorp\CoreBundle\Services\Lock\Interfaces\LockServiceInterface:
         class: LoyaltyCorp\CoreBundle\Services\Lock\LockService
+
+    # Custom Pagination start
+    LoyaltyCorp\CoreBundle\Pagination\CustomPaginationListener:
```

Comment:
We must be able to use this bundle in any Symfony app not only with Api platform, right now we can't because we will get errors because the Api platform services won't exist if not installed.

The service definitions for the Custom Paginator will need to be done in the extension and we must check if api platform is installed before defining them otherwise we don't.

Reply:
Done.